README.md

```diff
@@ -907,6 +907,28 @@ that file can be edited or just uploaded to [webtranslateit](http://docs.webtran
 }
 ```
 
+### Using a custom file format
+
+By default, `JSON.parse()` will be used to read files and `JSON.stringify()` will be used to write new strings to language files.
+
+It is, however, possible to specify your own parser, by setting `parser` and `reverseParser` keys in the `.configure` method. It's useful if you want to use other file formats, such as Ini files or YAML, instead of JSON, to write your language files. For example:
+
+```javascript
+i18n.configure({
+  extension: '.lang', // you'll probably not want to use .json
+  parser: function(fileContents) {
```

Comment:
naming `parser` as `reader`and `reverseParser` as `writer` sounds a bit more intuitive to me and more generic.  I could also imagine having some kind of database read/write passed into those options. What you thin?